specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2022-12-01/inference.json

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
What is this file? Shouldn't it be OpenAPI v2?

Reply:
This original OpenAI swagger is using OpenApi v3 because they use oneOf in the request schema.